modules/core/src/test/java/org/apache/ignite/internal/processors/database/CacheFreeListSelfTest.java

```diff
@@ -514,9 +514,12 @@ protected PageMemory createPageMemory(int pageSize, DataRegionConfiguration plcC
 
         IgniteConfiguration cfg = new IgniteConfiguration().setMetricExporterSpi(new NoopMetricExporterSpi());
 
-        DataRegionMetricsImpl regionMetrics = new DataRegionMetricsImpl(plcCfg,
-            new GridMetricManager(new GridTestKernalContext(new GridTestLog4jLogger(), cfg)),
-            NO_OP_METRICS);
+        GridTestKernalContext ctx = new GridTestKernalContext(new GridTestLog4jLogger(), cfg);
+
+        ctx.add(new GridMetricManager(ctx));
+
+        DataRegionMetricsImpl regionMetrics = new DataRegionMetricsImpl(plcCfg, ctx.metric(),
+            ctx.performanceStatistics(), NO_OP_METRICS);
```

Comment:
Should we add `PerformanceStatisticsProcessor` to the ctx first?

Reply:
done